ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlServerMain.java

```diff
@@ -46,6 +48,9 @@ public static void main(final String[] args) {
       );
 
       final String installDir = properties.getOrDefault("ksql.server.install.dir", "");
+      final String streamsStateDirPath = properties.getOrDefault("ksql.streams.state.dir",
```

Comment:
Replace `ksql.streams.state.dir` with `KsqlConfig.KSQL_STREAMS_PREFIX + StreamsConfig.STATE_DIR_CONFIG`?

Reply:
Yes, this one is much better. Updated!